systems/plants/ContactForceTorqueSensor.m

```diff
@@ -123,7 +123,7 @@
         nD = length(tsmanip.LCP_cache.data.contact_data.d);
         nC_body = nA + nB;
         nC = size(tsmanip.LCP_cache.data.contact_data.normal,2);
-        nL = manip.getNumJointLimitConstraints;
+        nL = sum(tsmanip.LCP_cache.data.possible_limit_indices);
         nP = 2*manip.num_position_constraints;  % number of position constraints
```

Comment:
the problem is that the unit test didn't have any position constraints.  it would be great to add one

- Russ

> On Feb 4, 2015, at 7:13 PM, Gregory Izatt <notifications@github.com> wrote:
> 
> In systems/plants/ContactForceTorqueSensor.m:
> 
> > @@ -123,7 +123,7 @@
> >          nD = length(tsmanip.LCP_cache.data.contact_data.d);
> >          nC_body = nA + nB;
> >          nC = size(tsmanip.LCP_cache.data.contact_data.normal,2);
> > -        nL = manip.getNumJointLimitConstraints;
> > +        nL = sum(tsmanip.LCP_cache.data.possible_limit_indices);e
> >          nP = 2*manip.num_position_constraints;  % number of position constraints
> Got it, no worries -- sounds like this hasn't been properly tested in a while. I can make the actual change whenever I'm next sitting down.
> …
> —
> Reply to this email directly or view it on GitHub.
>